sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/README.md

```diff
@@ -21,7 +21,7 @@ No package has been published for this client library yet.
 
 ### Authenticate the Client
 
-In order to interact with the Azure Time Series Insights service, you will need to create an instance of a [TokenCredential class][token_credential] and pass it to the constructor of your [TimeSeriesInsightsClient](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs).
+In order to interact with the Azure Time Series Insights service, you will need to create an instance of a [TokenCredential class][token_credential] and pass it to the constructor of your [TimeSeriesInsightsClient](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs).
```

Comment:
Are the folder paths being renamed also? This will break links if not

Reply:
I did change the folder name locally, but that change is not being reflected as part of the Git changes for some reason. Taking a look.

Reply:
cool, yeah I'll hold off on the other links until then.

Reply:
should be done now.